### DISCUSSION TITLE
Remove blue box outline in free text

### DISCUSSION THREAD

**Discussion by muhsaad47** [sumatrapdfreader/sumatrapdf#4101] [Q&A]
When I add a free text annotation a blue box outline appears. This wasnt present in previous versions. Is there anyway to remove it for a cleaner look?

**Comment by GitHubRulesOK** [comment]
it goes when the object is not in focus and its a way better solution than the old excessively thick black box. Agreed it is different to Acrobat
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/479abfd5-20fd-4781-a2e4-78117054e259)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/53dfc7b2-41e2-4ec0-ad44-99214633b534)
